src/Config/web.permissions.php

```diff
@@ -25,61 +25,166 @@
     'superuser',
     'queue_manager',
 
-    'apikey' => [
-        'delete',
-        'detail',
-        'toggle_status',
-        'list',
-        'update',
-    ],
-
     'character' => [
-        'assets',
-        'bookmarks',
-        'calendar',
-        'channels',
-        'contacts',
-        'contracts',
-        'fittings',
-        'industry',
-        'intel',
-        'jobs',     // Allows for the dispatching of jobs.
-        'killmails',
-        'list',
-        'mail',
-        'market',
-        'mining',
-        'notifications',
-        'pi',
-        'research',
-        'skills',
-        'standings',
-        'sheet',
-        'journal',
-        'transactions',
+        'asset' => [
+            'label' => 'Grant access to Character Assets',
```

Comment:
Translations :D

Reply:
Ah yeah, still on the TODO :D
I was waiting for user feedback, but I could probably send them by the meantime.